### DISCUSSION TITLE
Kimai desktop app - critical error openning a new time

### DISCUSSION THREAD

**Discussion by JaimeDuque2020** [kimai/kimai#3020] [Questions & Answers]
Hi,

Today some users report an error when they tried to open a new time record in the desktop app, the error it's only present in the desktop app, they can change/open a new time record in the web browser. however, a few minutes later the desktop app fixed itself and everything goes normally

Any clue about where the error could be?

![Kimai critical error](https://user-images.githubusercontent.com/83285737/145819806-e4d41c59-260a-4bf6-80f4-340ae1384fe8.png)

Best Regards,
Jaime Duque

**Comment by kevinpapst** [comment]
what desktop app?
